app/templates/patch.hbs

```diff
@@ -2,6 +2,7 @@
   <div class="file">
     <div class="meta">
       <span>{{filePath}}</span>
+      <a class="target" {{bind-attr href=targetFileUrl}} title="View generated file at target version">View</a>
```

Comment:
I took this both from a proposal and, from how GitHub does it (as in view file at revision). Probably, it is safe to assume that users will be upgrading more than downgrading. I would suggest that if two links are included, they should not have the same prominence.